src/app/queries/technicalRecord/axleQuery.ts

```diff
@@ -0,0 +1,25 @@
+const AXLE_QUERY = 'SELECT `a`.`id`, JSON_OBJECT('
+  + "'tyre', JSON_OBJECT("
```

Comment:
Reading the ticket it seems that the AC we have is very generic and will probably need some guidance around how we are going to test that or if we plan to add some automated tests around those.
As I understand, those queries will perform some data manipulation on existing record and provide a different output.
I am thinking that if we were to change this query it would be nice to have some safety net that tells us that something broke as it will interface with sql ?

Reply:
This is covered in our integration tests in the Java framework.